tests/storage/snapshot.go

```diff
@@ -1053,8 +1053,9 @@ var _ = SIGDescribe("VirtualMachineSnapshot Tests", func() {
 				origStatus := ss.Status
 				// zero out the times to be able to compare after
 				clearConditionsTimestamps(origStatus.Conditions)
-				ss.Status = nil
-				ss, err = virtClient.VirtualMachineSnapshot(ss.Namespace).Update(context.Background(), ss, metav1.UpdateOptions{})
+				patchPayload, err := patch.New(patch.WithRemove("/status")).GeneratePayload()
+				Expect(err).NotTo(HaveOccurred())
+				ss, err = virtClient.VirtualMachineSnapshot(ss.Namespace).Patch(context.Background(), ss.Name, types.JSONPatchType, patchPayload, metav1.PatchOptions{})
```

Comment:
So once the snapshot is ready, I would hope the status doesn't change
(So for example if some status field keeps hotlooping over itself, that's a bug)

We could also change this to an `UpdateStatus` call since we're not interested in any spec changes,
but as far as I can tell there should be no need to patch

Reply:
actually, scratch that, there is no status subresource for vmsnapshots